script/check-rust-version.py

```diff
@@ -0,0 +1,97 @@
+# Script to check that version of Rust in local toolchain and build scripts
+# matches upstream Signal library version.
+
+import urllib.request, urllib.error
+import subprocess
+import sys
+
+from typing import Optional, Tuple
+
+FILES = [
+    "build-wheels.sh",
+    ".circleci/config.yml",
+]
+
+UPSTREAM_TOOLCHAIN_FILE_URL = (
+    "https://raw.githubusercontent.com/signalapp/libsignal-client/master/rust-toolchain"
```

Comment:
so as background/context right now I've been building these bindings against an [older rev on a fork](https://github.com/freedomofpress/libsignal-client/tree/build) (just for stability since the upstream project is in active development). Every so often we'd want to update both the fork and these bindings to track upstream. 

However regardless I think this script could be useful so I'm merging. Thanks!